api/helm/api/values.yaml

```diff
@@ -16,7 +16,7 @@ php:
   env: prod
   debug: '0'
   secret: ""
-  corsAllowOrigin: "^https?://.*?\\.api-platform\\.com$"
+  corsAllowOrigin: "^https?://(.*?api-platform\\.com|localhost(?::[0-9]+)?)$"
```

Comment:
Now `fakeapi-platform.com` is allowed! :stuck_out_tongue_closed_eyes:

Reply:
Now? It was already possible.

Reply:
No it wasn't because of the `\.` part. Indeed we must add it back.

Reply:
Fixed regex: https://regex101.com/r/EA7Gzu/3

Reply:
We don't call the demo from `api-platform.com`, so https://regex101.com/r/EA7Gzu/5 is enough.

Reply:
Better: https://regex101.com/r/EA7Gzu/6